test/console/command_index_test.go

```diff
@@ -28,22 +28,66 @@ func Test_index_with_one_command(t *testing.T) {
 	require.Contains(
 		t,
 		TrimDoubleSpaces(output.String()),
-		"Confetti (testing)\x1b[39m" +
-			"\n\n" +
-			" -h --help Can be used with any command to show\n" +
-			" the command's available arguments and options.\n\n" +
-			" baker Interact with your application.\n" +
+		"Confetti (testing)\x1b[39m"+
+			"\n\n"+
+			" -h --help Can be used with any command to show\n"+
+			" the command's available arguments and options.\n\n"+
+			" baker Interact with your application.\n"+
 			" log:clear Clear the log files as indicated in the configuration.",
 	)
 }
 
+func Test_command_suggestions_on_failed_command(t *testing.T) {
+	output, app := setUp()
+	kernel := console.Kernel{
+		App:      app,
+		Writer:   &output,
```

Comment:
I never actually use Run(). What are the advantages? Is it true that 2 Runs in 1 test function can run in parallel? If you have a global writer, do the 2 tests write on 1 writer?

Reply:
`t.Run()` does exactly the same as if you would write completely independent functions. I like to use it if there are many tests and I need to structure my tests.
Further, as you mentioned, you can set some variables globally for all t.Run in the same test function.